### PR TITLE
Add --spec option to run a single spec file

### PR DESCRIPTION
**Pull request by cristianbote**
**What**:

I've added a `--spec` option for our e2e script in order to be able to pass the name for the spec file we would like to run.

**Why**:

While debugging tests, locally, we need a way to only run, in isolation, one spec file.

**How**:

Added a `--spec` option that overrides an already provided suite option.

**Tasks**:

- [x] End to end tests

**Unrelated Tasks**

- [ ] Code
- [ ] TSDocs
- [ ] TypeScript
- [ ] Unit tests
- [ ] TypeScript tests
- [ ] Update starter themes
- [ ] Update other packages
- [ ] Open an issue for this feature in the [docs repo](https://github.com/frontity/docs/wiki/Code-Releases)
- [ ] Update community discussions
- [ ] Add a changeset (with link to its [Feature Discussion](https://community.frontity.org/c/33) if it exists)

**Additional Comments**

### CODE REVIEW

**Comment by cristianbote** [comment]
@luisherranz we've talked about this one before 😃  let me know what you think. 👍

File: e2e/e2e.js [hunk 1]
```diff
@@ -170,7 +169,9 @@ process.env["CYPRESS_FRONTITY_MODE"] =
           await cypress.run({
             env: { WORDPRESS_VERSION: wpVersion },
             browser,
-            spec: `./integration/${suite}/**/*.spec.js`,
+            spec: spec
+              ? `./integration/**/**/${spec}.spec.js`
```
Comment (luisherranz) [inline e2e/e2e.js]: This double globstar is not required.

```suggestion
              ? `./integration/**/${spec}.spec.js`
```

**Review by luisherranz — APPROVED** [review]
It's good enough for now! Thanks Cris 😄 

_Although I may review how it works if we finally need to use it to launch specific specs in the BrowserStack integration._

**Comment by cristianbote** [comment]
Thank you @luisherranz.

**Pull request merged**